pkg/notify/elasticsearch.go

```diff
@@ -61,7 +62,13 @@ func NewElasticSearch(c config.ElasticSearch) (Notifier, error) {
 	if c.AWSSigning.Enabled {
 		// Get credentials from environment variables and create the AWS Signature Version 4 signer
 		sess := session.Must(session.NewSession())
-		if c.AWSSigning.RoleArn != "" {
+
+		// Use OIDC token to generate credentials if using IAM to Service Account
+		awsRoleARN := os.Getenv("AWS_ROLE_ARN")
+		awsWebIdentityTokenFile := os.Getenv("AWS_WEB_IDENTITY_TOKEN_FILE")
```

Comment:
Please declare "AWS_ROLE_ARN" and "AWS_WEB_IDENTITY_TOKEN_FILE" as consts

Reply:
These are the environment variables which will be injected to the pod when we use the eks webhook identity annotation.

Reply:
Done